fyde-app/linux.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: Linux
+parent: Fyde App
+nav_order: 1
+---
+# Fyde App - Linux
+
+Instructions to install the Fyde App in different distributions.
+
+There are two versions available:
+
+- fyde - complete GUI version (includes fyde-tunnel)
+- fyde-tunnel - CLI version
+
+Supported versions:
+
+| Distro    | Versions      | Instructions              |
+| ---       | ---           | ---                       |
+| Arch      | -             | In progress               |
+| CentOS    | 7, 8          | [YUM](##yum-repository)   |
+| Debian    | 9, 10         | [APT](##apt-repository)   |
+| Linux     | x86_64        | In progress               |
```

Comment:
I would write "Other Distribution" and place it in the end of the table.

Reply:
What about this change @GMartinez-Sisti

Reply:
That's what you get for multitasking. Updated.